pyat/at/collective/wake_object.py

```diff
@@ -165,3 +165,49 @@ def reswall(self, wcomp, length, rvac, conduct, beta, yokoya_factor=1):
                                       length, rvac, conduct, beta)
         else:
             raise AtError('Invalid WakeComponent: {}'.format(wcomp))
+
+
+def res_object(srange, wakecomp, frequency, qfactor, rshunt, beta,
+              yokoya_factor=1):
+    """
+    Method to build a resonator wake object
+    """
+    wake = Wake(srange)
+    wakecomp = numpy.atleast_1d(wakecomp)
+    try:
+        frequency = numpy.broadcast_to(frequency, (len(wakecomp), ))
+        qfactor = numpy.broadcast_to(qfactor, (len(wakecomp), ))
+        rshunt = numpy.broadcast_to(rshunt, (len(wakecomp), ))
+        yokoya_factor = numpy.broadcast_to(yokoya_factor, (len(wakecomp), )) 
+    except ValueError:
+        raise AtError('Wake object inputs should be either scalars '
+                      'or with shape (len(wakecomp), )')
+    for wc, fr, qf, rs, yk in zip(wakecomp, frequency, qfactor, rshunt, yokoya_factor):
+        wake.add(WakeType.RESONATOR, wc, fr, qf, rs, beta, yokoya_factor=yk)
+    return wake
+
+
+def longres_object(srange, frequency, qfactor, rshunt, beta):
```

Comment:
We have res which is short for resonator, then reswall which is short for resistive. I propose to change reswall_object to rw_object